internal/provider/resource_consul_cluster_test.go

```diff
@@ -23,7 +23,7 @@ var updatedConsulCluster = `
 resource "hcp_consul_cluster" "test" {
 	cluster_id = "test-consul-cluster"
 	hvn_id     = hcp_hvn.test.hvn_id
-	tier       = "standard"
+	tier       = "development"
 	size	   = "medium"
```

Comment:
With a dev tier shouldn't size be xsmall?

Reply:
Makes sense, this was a symptom of me being impatient with the ~35 minute acc testing. I'll fix and run those again